### PR TITLE
PyPI formatting

### PR DESCRIPTION
**Pull request by SimonBiggs**
Update pyproject.toml to include the various tags expected by PyPI. We now have the `rai` package name on PyPI! :slightly_smiling_face: 

https://pypi.org/project/rai/

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -22,16 +22,12 @@ achieve.
 
 ## Installation
 
-The aim is for you to be able to install this open source research library by
-running:
+This open source research library is abel to be installed by running:
```
Comment (sjswerdloff) [inline README.md]: typo: able

Comment (SimonBiggs) [inline README.md:25]:
```suggestion
This open source research library is able to be installed by running:
```

**Review by sjswerdloff — CHANGES_REQUESTED** [review]
it's a nit, but for a README, spelling is critical to provide a professional appearance

**Review by sjswerdloff — APPROVED** [review]

File: README.md (hunk 1 above)
Comment (SimonBiggs) [reply README.md]: > but for a README, spelling is critical to provide a professional appearance

Yup, I completely agree. Thanks Stuart :slightly_smiling_face:. Misspelling, especially in a document like that comes off quite unprofessional.

**Pull request merged**